python_ta/cfg/visitor.py

```diff
@@ -33,19 +35,42 @@ def visit_generic(self, node: NodeNG) -> None:
         self._current_block.add_statement(node)
 
     def visit_module(self, module: astroid.Module) -> None:
-        self.cfg = ControlFlowGraph()
-        self._current_block = self.cfg.start
+        self.cfgs.append(ControlFlowGraph())
+        self._current_cfg = self.cfgs[0]
+        self._current_block = self._current_cfg.start
 
         for child in module.body:
             child.accept(self)
-        self.cfg.link_or_merge(self._current_block, self.cfg.end)
+
+        self._current_cfg.link_or_merge(self._current_block, self._current_cfg.end)
+
+    def visit_functiondef(self, func: astroid.FunctionDef) -> None:
+        previous_cfg = self._current_cfg
+        previous_block = self._current_block
+        previous_block.add_statement(func)
```

Comment:
Do this line at the very top of the function (`self._current_block.add_statement`) to distinguish it from the rest of the function.